src/olympia/editors/tests/test_views.py

```diff
@@ -2490,6 +2490,60 @@ def test_viewing(self):
         assert data['current_name'] == self.editor.name
         assert data['is_user'] == 0
 
+    def test_viewing_lock_limit(self):
+        # Let just override this to make the test a bit shorter.
+        amo.EDITOR_REVIEW_LOCK_LIMIT = 1
```

Comment:
it's best to actually use `mock.patch` for that since you're effectively changing that value now for all other tests too (theoretically you could remove it from one of the both tests and they would still succeed depending on execution order).